bin/templates/cordova/Api.js

```diff
@@ -51,7 +51,7 @@ function setupEvents (externalEventEmitter) {
 
 class Api {
     constructor (platform, platformRootDir, events) {
-        this.platform = 'electron';
+        this.platform = 'cordova-electron';
```

Comment:
https://github.com/apache/cordova-ios/blob/master/bin/templates/scripts/cordova/Api.js#L59
https://github.com/apache/cordova-android/blob/master/bin/templates/cordova/Api.js#L56

ios api reports 'ios'
android api reports 'android'

Reply:
In current PR it is `this.platform = 'electron';`.

I am guessing you might be looking at each individual commits where at one point it was changed to 'cordova-electron'. It was changed previously because `cordova-lib` didn't understand that cordova-electron was electron. 

When adding the platform from GH or local cloned repo, the platform folder and config.xml entry uses `cordova-electron`.

That means commands like `cordova run electron` and `cordova platform rm electron` will not work. Must use `cordova run cordova-electron` etc.

Cordova-lib is not updated but is expected to be before this PR.. if we want to make this PR work without modifying lib, I recommend these changes 

https://github.com/erisu/cordova-electron/compare/electron-improvements...erisu:electron-improvements-no-lib

But we must use `cordova run cordova-electron`